src/test/resources/requests/user/list-existent-users.json

```diff
@@ -0,0 +1,44 @@
+[
+  {
+    "id": "6529ef53-3551-4811-bc88-d755abad5531",
+    "name": "user1",
+    "preferredEmail": "user1@mail.com",
+    "enabled": false,
+    "validated": false,
+    "roles": [
```

Comment:
This object will change after the work of @k2rosvall .. so let's wait till it is done to merge this one please